### ISSUE TITLE
Django rest framework and normalizr 

### ISSUE THREAD

**Issue by pcompassion** [paularmstrong/normalizr#165]
For a following schema (expressed in django models)

    class A:
      b = foreigneKey(B)
      
    
    class B:
      pass

Django rest framework serializes A instance as

    { 
      'id': 1,
      'b': 3
    }

normalizr seems to want a format of

    {
      'id': 1,
      'b': {
        'id': 3
      }
    }

I could change DRF 's default PrimaryKeyRelatedField's serialization, but I 'm wondering if there's another way?  
Maybe there's a way to tell normalizr that 'b' is itself an id attribute?

**Comment by paularmstrong** [comment]
It looks like your data is already normalized. Maybe you don't need to use normalizr.

**Comment by pcompassion** [comment]
ah, I'll make it more clear.

it may look like it's normalized because an entity is represented as an integer key.  
I guess you can say it is.  

Django rest framework serializes instance A when I only need B's key

```
{
  'id': 1,
  'b': 3
}
```

or when I need B's key and its associated data

```
{
  'id': 1,
  'b': {
    'id': 3
    'name': 'some'
  }
}
```

So it's just Django rest framework's default serialization, and I still want to use normalizr for both cases because it's consistent doing so.

**Comment by paularmstrong** [comment]
This should work in [v3.0.0](https://github.com/paularmstrong/normalizr/tree/normalizr3), which is available for testing from npm as v3.0.0-alpha.5
